Library/ViewModels/PledgePaymentMethodsViewModel.swift

```diff
@@ -198,22 +207,26 @@ public final class PledgePaymentMethodsViewModel: PledgePaymentMethodsViewModelT
 
         if indexPath.row < paymentSheetPaymentMethodCount {
           // we are selecting a new payment sheet card
-          let setupIntent = data.paymentSheetPaymentMethodsCellData[indexPath.row].setupIntent
+          let clientSecret = data.paymentSheetPaymentMethodsCellData[indexPath.row].clientSecret
```

Comment:
At the level of each individual payment row, it just tracks a `clientSecret` as a string. The `PledgePaymentMethodsViewModel`, one level up, keeps track of whether that string is a `PaymentIntent` or a `SetupIntent`.